.circleci/config.yml

```diff
@@ -113,6 +113,18 @@ commands:
         type: boolean
         default: false
     steps:
+      # CircleCI images fail on Ruby 3.3
+      # https://github.com/fastlane/fastlane/issues/21794#issuecomment-2021331335
+      - run:
+          name: Set Ruby 3.2.2
+          command: |
+            eval "$(rbenv init -)"
+            rbenv install -s 3.2.2  # Replace with your required Ruby version
+            rbenv global 3.2.2
+            rbenv rehash
```

Comment:
I was seeing the merged PR, did you try with `switch-ruby` from the macos orb? https://circleci.com/developer/orbs/orb/circleci/macos#commands-switch-ruby

Reply:
I did! It didn't work for me 😬 I was tried switching to `3..2.2` with that and also printed the version after and it just didn't take which I thought was really wweird 🤷‍♂️